worlds/sa2b/CHANGELOG.md

```diff
@@ -0,0 +1,247 @@
+# Sonic Adventure 2 Battle - Changelog
+
+
+## v2.3 - The Chao Update
+
+### Features:
+
+- New goal
+	- Chaos Chao
+		- Raise a Chaos Chao to win!
+- New optional Location Checks
+	- Chao Animal Parts
+		- Each body part from each type of animal is a location
+	- Chao Stats
+		- 0-99 levels of each of the 7 Chao stats can be locations
+		- The frequency of Chao Stat locations can be set (every level, every 2nd level, etc)
+	- Kindergartensanity
+		- Classroom lessons are locations
+		- Either all lessons or any one of each category can be set as locations
+	- Shopsanity
+		- A specified number of locations can be placed in the Chao Black Market
+		- These locations are unlocked by acquiring Chao Coins
+		- Ring costs for these items can be adjusted
+	- Chao Karate can now be set to one location per fight, instead of one per tournament
+- Items
+	- If any Chao locations are active, the following will be in the item pool:
+		- Chao Eggs
+		- Garden Seeds
+		- Garden Fruit
+		- Chao Hats
+		- Chaos Drives
+- The starting eggs in the garden can be a random color
```

Comment:
If you want, you could group these three into their own category of "Miscellaneous" or something like that since everything else is under a category, but it's up to you